app/src/geckoBeta/java/org/mozilla/fenix/engine/GeckoProvider.kt

```diff
@@ -35,6 +36,7 @@ object GeckoProvider {
             .crashHandler(CrashHandlerService::class.java)
             .useContentProcessHint(true)
             .telemetryDelegate(GeckoAdapter())
+            .debugLogging(BuildConfig.DEBUG)
```

Comment:
This adds debug logging to all gecko beta builds. Since gv beta is currently used for release, wouldn't this enable debug logging on release builds as well?

Reply:
Same thing below too.

Reply:
I'm confused. Is `BuildConfig.DEBUG` not doing what I think it's doing? What I'm trying to do is set the `debugLogging` bool based on the Fenix BuildConfig Debug bool

Reply:
Oh! My misunderstanding, sorry! Yes that's right. :)

I read that as "set logging level to debug".